app/models/docker_builder_service.rb

```diff
@@ -60,6 +61,7 @@ def push_image(tag)
         build.docker_repo_digest = "#{project.docker_repo}@#{matches[1]}"
       end
     end
+    build.docker_repo_digest ||= result.info['RepoTags'].first
```

Comment:
@henders, was trying to understand this line. So, as far as I understood, when we upload a docker image, we keep it's digest in the corresponding build. So, in this case, when we're not able to get the digest from the response, what's the meaning of that `RepoTag`? Does it contain a digest of the image, as well, or is it just a reference to identify that image?

The other thing I did not understand is how do we know this `RepoTags`is returned and what does it mean? Was trying to find documentation in the Docker API v1.16, but could only find this info in the List API. :)